### ISSUE TITLE
Does `--fixed_curvature=True` work?

### ISSUE THREAD

**Issue by macio232** [oskopek/mvae#3]
**Describe the bug**
Despite setting the `--fixed_curvature=True` flag I can see that curvature is being changed during training.

**To Reproduce**
1. Set up your environment like described in README
2. Run experiment with `python -m mt.examples.run --dataset="mnist" --model="s10" --fixed_curvature=True ` 
3. Observe `comp_000_s10/curvature` is being changed

**Expected behavior**
Curvature should be constant and set to the default value, which is `1`.

**Desktop (please complete the following information):**
 - OS: Linux

**Comment by macio232** [comment]
Ok, I can see the reason
https://github.com/oskopek/mvae/blob/96d851ce446d8f7e975d82ec74caabb67cfbcd77/mt/mvae/models/train.py#L179-L184

And I found a reference for such behavior in the paper
> The first 10 epochs for all models are trained with a fixed curvature starting at 0 and increasing in absolute value each epoch.

In my experiments, I have skipped the burn-in phase what resulted in AssestrionError in `assert torch.isfinite(z_mean).all()` in `encode` of `component`.

**Comment by oskopek** [comment]
Yes, stability is a big reason why we did the burn in. If you want, you can try ignoring assertions by adding the `-O` (big letter O) flag to python when running training.

Any other stability-related changes might also help, like lowering the learning rate, etc.
